lua/cmp/context.lua

```diff
@@ -55,8 +55,9 @@ context.new = function(prev_context, option)
   self.cursor_line = vim.api.nvim_get_current_line()
   self.virtcol = vim.fn.virtcol('.')
   self.cursor = {}
-  self.cursor.row = vim.api.nvim_win_get_cursor(0)[1]
-  self.cursor.col = vim.api.nvim_win_get_cursor(0)[2] + 1
+  local cursor = vim.api.nvim_win_get_cursor(0)
```

Comment:
Here, calling `nvim_win_get_cursor`once is more efficient than calling twice. Otherwise I made no change.